go.mod

```diff
@@ -6,7 +6,7 @@
 
 module k8s.io/kubernetes
 
-go 1.22.0
+go 1.21
```

Comment:
-1 on this... the go version used to build kubernetes doesn't impact the go version used to build the runc binary

We want to stay on latest go in latest kube releases (in this case, the loop var behavior and go workspaces are things we care about using go 1.22 with)

Reply:
It also requires more changes through the codebase as it seems.

Reply:
Yeah, the runc refusal to build with go1.22 looks self-inflicted, and limited to the nsenter package which kubernetes does not vendor / build / link.